### ISSUE TITLE
Feros: Geth in the Tunnels is uncompletable if the transmitter is destroyed by tick damage

### ISSUE THREAD

**Issue by henbagle** [henbagle/LE1CommunityPatch#22]
**Describe the bug**
Feros: Geth in the Tunnels - This quest will be uncompletable if the transmitter is destroyed by tick damage (such as warp or environmental hazards)

**Comment by henbagle** [comment]
Either was never a bug or was fixed by BioWare. After testing, the quest is completable after destroying the transmitter via Warp tick damage.

**Comment by henbagle** [comment]
UPDATE: This is actually a bug, and has now been fixed. Warp tick damage doesn't cause the bug, but incendiary ammo does.

File: BIOA_WAR20_12a_DSG.pcc

The BioSeqEvt_TakeDamage sequence object doesn't get triggered by incendiary ammo's tick damage. To fix this, we use a ArtPlaceableTransition to check when the transmitter's state gets set to "Destroyed", and then trigger the existing logic to update the journal. I've added a self-closing gate to retain the original logic of checking the transmitter's health, so whichever method returns true first will update the quest. Exports 3229-3233 in sequence 3087.
